src/host/exe/exemain.cpp

```diff
@@ -5,6 +5,7 @@
 
 #include "ConsoleArguments.hpp"

```

Comment:
renaming this file will need you to update `sources`.

Reply:
done